.gitignore

```diff
@@ -19,8 +19,11 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-#vscode files
-.vscode
+# Ignore package tarball (built via "mix hex.build").
+ueberauth_microsoft-*.tar
+
+# Temporary files, for example, from tests.
+/tmp/
 
-# ignore elixir language server files
-/.elixir_ls
```

Comment:
Looks like the `/.elixir_ls` got removed. Could you add that back in please as it prevents files being committed that are generated by running the language server.

Reply:
@swelham Elixir LS already self-ignored its own folder?

```
$ cat .elixir_ls/.gitignore 
*
```

Reply:
Ah, I didn't realise that 👍